### PR TITLE
Merge pull request #1 from MikitaTsiarentsyeu/main

### PR DESCRIPTION
**Pull request by RomanPutsilouski**
task

### CODE REVIEW

**Comment by RomanPutsilouski** [comment]
/

**Review by MikitaTsiarentsyeu — CHANGES_REQUESTED** [review]
image

It's not working, and it's a mess. I strongly recommend remove all unnecessary comments (including snippet formatting like # ______________Введение времени с терминала______________) and execute both current time and user value one after another, rather then commenting one of it, it's really hard to read your code with such formatting.

**Comment by RomanPutsilouski** [comment]
Добработал все+сортировку добавил

**Comment by RomanPutsilouski** [comment]
finished

File: Tasks/Putsilouski_Tasks/CW/1/claswork.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+с = "five thirteen two eleven seventeen two one thirteen ten four eight five nineteen"
+c = с.split(" ")
+for i in range(len(c)):
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Putsilouski_Tasks/CW/1/claswork.py:3]: and technically it's a new variable, you're supposed to use c in every loop (God knows how)

File: Tasks/Putsilouski_Tasks/Task2/HW2new.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+f_line = input("Введи время,:\n")
+s_line = f_line.replace(' ', '')
+if ':' not in s_line:
+    sys.exit("Неверный формат, начни заново")
+t_line = s_line.split(":")
+h_с = int(t_line[0])
+m_с = int(t_line[1])
+if h_с < 0 or h_с > 24 or m_с < 0 or m_с > 59:
+    sys.exit("Неверный формат, начни заново")
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Putsilouski_Tasks/Task2/HW2new.py:78]: good validation

**Review by MikitaTsiarentsyeu — APPROVED** [review]
I score your homework at 8.5, classwork - at 9.0.

**Pull request merged**